build.gradle.kts

```diff
@@ -1,9 +1,49 @@
+import net.pwall.json.kotlin.codegen.gradle.JSONSchemaCodegenPlugin
+import net.pwall.json.kotlin.codegen.gradle.JSONSchemaCodegenTask
+
+buildscript {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+    }
+    dependencies {
+        // A.t.m. we need our patched version of json-kotlin-schema-codegen here, locally supplied via "mvn install".
+        classpath("net.pwall.json:json-kotlin-schema-codegen:0.108")
+        classpath("net.pwall.json:json-kotlin-gradle:0.107")
+    }
+}
+
 plugins {
     id("buildlogic.kotlin-common-conventions")
 }
 
 dependencies {
     // This merges all our individual kover results into the root project
     kover(project(":csaf-import"))
-    kover(project(":csaf-schema-codegen"))
-}
\ No newline at end of file
+}
+
+// Create and register ExecutionService which enforces serial execution of assigned tasks.
```

Comment:
Is this needed? unnecessary c/p

Reply:
Will be needed for later publishing, I guess.

Reply:
ok